CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## unreleased
 
 * [CHANGE] Rules: Execute 1 query instead of N (where N is the number of alerts within alert rule) when restoring alerts. #13980
+* [FEATURE] Ruler: Allow setting `evaluation_delay` for each rule group via rule group configuration file and globally. #14061
```

Comment:
Ruler is a Cortex/Mimir terminology, in Prometheus I'd just write "Rules" here, like in the item above.

Reply:
Done in https://github.com/prometheus/prometheus/pull/14061/commits/118af0580957162aeecbfbd8d7f51724a099203b